layout-manager/pkg/mcp/handlers.go

```diff
@@ -812,6 +812,68 @@ func GetActiveDashboardHandler(layoutRepo *db.LayoutRepository) func(context.Con
 	}
 }
 
+
+// SetDashboardMetadataHandler handles the set_dashboard_metadata tool
+func SetDashboardMetadataHandler(layoutRepo *db.LayoutRepository) func(context.Context, mcp.CallToolRequest) (*mcp.CallToolResult, error) {
+    return func(ctx context.Context, request mcp.CallToolRequest) (*mcp.CallToolResult, error) {
+        log.Printf("[MCP] set_dashboard_metadata called")
+
+        layoutID := request.GetString("layout_id", "")
+        if layoutID == "" {
+            layoutID = request.GetString("dashboard_id", "")
+        }
+        dashboardName := request.GetString("dashboard_name", request.GetString("dashboard", ""))
+
+        newName := request.GetString("name", "")
+        newDescription := request.GetString("description", "")
+
+        var layout *models.Layout
+        var err error
+        switch {
+        case layoutID != "":
+            layout, err = layoutRepo.GetByLayoutID(layoutID)
+        case dashboardName != "":
+            layout, err = layoutRepo.GetByNameInsensitive(dashboardName)
```

Comment:
This method is missing from the repository so my layout-manager fails to start. Can you check if you forgot to commit it? It would be in `repository.go`

Reply:
ended up removing these so marking this convo as resolved